### ISSUE TITLE
Missing multinode integration test

### ISSUE THREAD

**Issue by toliaqat** [Agoric/Agoric#9613]
## What is the Problem Being Solved?
We still don't have a diverse multinode integration test. A3p integration tests are single-node tests.

## Description of the Design
Make a3p run in multinode (multiple validators), this will require changes to how docker images is build with backed in state. 

## Security Considerations
NA

**Comment by mhofman** [comment]
This multi-node test should also exercise state-sync export and restore like the current `deployment-test` does.

Also something to keep in mind is that currently any docker based test (like a3p integration) does not respect endo integration: https://github.com/Agoric/agoric-sdk/issues/8938. The current deployment-test does behave correctly here I believe, and we should try to not regress.

**Comment by dckc** [comment]
Should this reduce the need to use loadgen, and hence address issues such as this one?
 - https://github.com/Agoric/testnet-load-generator/issues/115

**Comment by mhofman** [comment]
Yes this is the issue about replacing the deployment / loadgen test.

**Comment by mhofman** [comment]
Thought about this some more, and I think a single validator node + a loadgen follower started from state-sync and running with xsnap random init should be sufficient to test determinism regressions. We may want to configure said follower to not restart workers on snapshots taken to tickle the beast a little more.
